applications/RomApplication/python_scripts/hrom_training_utility.py

```diff
@@ -212,16 +213,16 @@ def AppendHRomWeightsToRomParameters(self):
         if self.hrom_output_format=="numpy":
             element_indexes = np.where( indexes < number_of_elements )[0]
             condition_indexes = np.where( indexes >= number_of_elements )[0]
-            np.save('HROM_ElementWeights.npy',weights[element_indexes])
-            np.save('HROM_ConditionWeights.npy',weights[condition_indexes])
-            np.save('HROM_ElementIds.npy',indexes[element_indexes]) #FIXME fix the -1 in the indexes of numpy and ids of Kratos
-            np.save('HROM_ConditionIds.npy',indexes[condition_indexes]-number_of_elements) #FIXME fix the -1 in the indexes of numpy and ids of Kratos
+            np.save(f'numpy_rom_data_{self.rom_basis_output_name}/HROM_ElementWeights.npy',weights[element_indexes])
```

Comment:
When comparing with the json formatting above I realized that they behave slightly different. 

In the json case, we are saving the ROM basis in the current work directory. However, in the numpy case the files are always saved in a folder. On top, in the json case the `rom_basis_output_name` refers to the filename (which I think is what the user would expect). However, in the npy case this refers to the folder case (which is not expected by the user) but the filenames are hardcoded.

At this point, maybe the easiest solution is to always save the ROM basis in a folder according to a `rom_basis_output_folder` and prepend the filenames with the content of `rom_basis_output_name` in the case of HROM npy output.

Reply:
Makes sense for me.